### PR TITLE
Add methods for creating a dir, getting current dir, create a new fil…

### PR DESCRIPTION
**Pull request by VitaliyKrivoshchapov**
…e, get list of  all nested dir

### CODE REVIEW

File: searchup/blocks/djfedos_db/test_CollectionsManager.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import pathlib
+
+from searchup.blocks.djfedos_db.fapi_server import app
+
+
+def test_add_collection():
+    from fastapi import FastAPI
+    from fastapi.testclient import TestClient
```
Comment (yairdardev) [inline searchup/blocks/djfedos_db/test_CollectionsManager.py:8]: not in use remove

File: searchup/blocks/djfedos_db/test_web_server_tokens.py [hunk 2]
```diff
@@ -101,3 +101,4 @@ def test_get_suggestions():
     assert data["limit"] == 10
     assert data["result"] == ['symptomatic']
 
+# add get
```
Comment (yairdardev) [inline searchup/blocks/djfedos_db/test_web_server_tokens.py:104]: @VitaliyKrivoshchapov 
its good you have added tests for implementation, 
but you need to add Please add Tests for API,

using TestClient

see examples in this file
```
 client = TestClient(app)
response = client.get("/get_suggestions/" + prefix)
```

you need to test next endpoints:
```python
@app.post("/collections/set_item/{item_name_id}")
def set_collection_item(item_name_id):
  ...

@app.get("/collections/get_item/{item_name_id}")
def get_collection_item(item_name_id):
   ...

@app.get("/collections/list")
def list_collections(prefix):
   ...
```

**Review by yairdardev — CHANGES_REQUESTED** [review]
please add tests for API 
- /collections/set_item/
- /collections/get_item/
- /collections/list
see #4

File: searchup/blocks/djfedos_db/CollectionsManager.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        p = Path(name)  # correct path
+        #(p / f"{kind}.txt").write_text("")
+
+
+    def get_collection(self, name):
+        p = Path("__wdir", name)
+        #print(p)
+        if p.exists():
+            print("Exists", p)
+            return p
```
Comment (yairdardev) [inline searchup/blocks/djfedos_db/CollectionsManager.py:22]: p.name

File: searchup/blocks/djfedos_db/CollectionsManager.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+class CollectionsManager:
+
+    def __init__(self, path_to_wdir = Path.cwd()):
+        self.wdir_p = Path(path_to_wdir)
+        self.wdir_p.mkdir(exist_ok=True, parents=True)
+
+    def add_collection(self, name, kind='tokens_trie'):# create new folder if nor exist
+        Path(self.wdir_p, name).mkdir(exist_ok=True, parents=True)
+        p = Path(name)  # correct path
+        #(p / f"{kind}.txt").write_text("")
```
Comment (yairdardev) [inline searchup/blocks/djfedos_db/CollectionsManager.py:14]: return p.name

File: searchup/blocks/djfedos_db/CollectionsManager.py [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+
+def test_add_collection(tmpdir):
+    # arrange
+    col = CollectionsManager(tmpdir)
+    name = "tmp_dir_test"
+    p = Path(tmpdir, name)
+    # act
+    # print(" print path", p)
+    # print("print tmpdir", tmpdir)
```
Comment (yairdardev) [inline searchup/blocks/djfedos_db/CollectionsManager.py:52]: remove comments

**Comment by yairdardev** [comment]
@lioryah

File: searchup/blocks/djfedos_db/CollectionsManager.py (hunk 3 above)
Comment (djfedos) [reply searchup/blocks/djfedos_db/CollectionsManager.py]: fixed, thanks!

File: searchup/blocks/djfedos_db/CollectionsManager.py (hunk 4 above)
Comment (djfedos) [reply searchup/blocks/djfedos_db/CollectionsManager.py]: fixed, thanks!

File: searchup/blocks/djfedos_db/CollectionsManager.py (hunk 5 above)
Comment (djfedos) [reply searchup/blocks/djfedos_db/CollectionsManager.py]: fixed, thanks!

File: searchup/blocks/djfedos_db/test_CollectionsManager.py (hunk 1 above)
Comment (djfedos) [reply searchup/blocks/djfedos_db/test_CollectionsManager.py:8]: removed, thanks!

File: searchup/blocks/djfedos_db/fapi_server.py [hunk 6]
```diff
@@ -152,9 +162,32 @@ def search_in_one_collection(item_name_id, prefix, limit: Optional[int] = 10):
     return {"we search in ONE collection now. prefix": prefix}
 
 
+
+
 def main():
     uvicorn.run(app, host="0.0.0.0", port=18000)
 
+# create a new director
```
Comment (djfedos) [inline searchup/blocks/djfedos_db/fapi_server.py:170]: this is already in CollectionManager, remove lines 170 to 190

File: searchup/blocks/vit_db/creat_db.py [hunk 7]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (djfedos) [inline searchup/blocks/vit_db/creat_db.py:1]: Please remove all the PostgreSQL related code from this branch

**Review by djfedos — CHANGES_REQUESTED** [review]

**Comment by yairdardev** [comment]
@VitaliyKrivoshchapov please close it or fix and merge

**Pull request closed without merging**